code/README.md

```diff
@@ -1,131 +1,221 @@
+ 2. [Git](https://git-scm.com/).
+ 3. Building the libiamf requires dependent audio codec libraries: [opus](https://downloads.xiph.org/releases/opus/opus-1.4.tar.gz) [fdk-aac](https://people.freedesktop.org/~wtay/fdk-aac-free-2.0.0.tar.gz) [flac](https://downloads.xiph.org/releases/flac/flac-1.4.2.tar.xz)
+ 4. Enabling the binaural rendering in libiamf requires external libraries: [bear](https://github.com/ebu/bear) [resonance-audio](https://github.com/resonance-audio/resonance-audio)
+ 
+### Get the code
+
+The IAMF library source code is stored in the Alliance for Open Media Git
+repository:
+
+~~~
+    $ git clone https://github.com/AOMediaCodec/libiamf
+    # By default, the above command stores the source in the libiamf/code directory:
+    $ cd libiamf/code
+~~~
+
+### Basic build
+
+"build.sh" is an example to build, you can run it directly at your side.  
+(dependent codec libraries and external libriaries complied under x64 linux have been provided in advance)
+
+CMake replaces the configure step typical of many projects. Running CMake will
+produce configuration and build files for the currently selected CMake
+generator. For most systems the default generator is Unix Makefiles. The basic
+form of a makefile build is the following:
+
+~~~
+    $ cmake .
+    $ make
+~~~
+
+### Configuration options
+
+The IAMF library has few configuration options, There are one option which is used to enable binaural rendering:
+    Build binaural rendering configuration options. These have the form `BINAURALIZER`.
+~~~
+    $ cmake ./ -DBINAURALIZER=ON
+    $ make
+~~~
+
+### Dylib builds
+
+A dylib (shared object) build of the IAMF library is enabled by default.
+~~~
+    $ cmake .
+    $ make
+~~~
+
+### Cross compiling
+
+For the purposes of building the IAMF and applications and relative to the
+scope of this guide, all builds for architectures differing from the native host
+architecture will be considered cross compiles. The IAMF CMake build handles
+cross compiling via the use of toolchain files included in the AV1 repository.
+The toolchain files available at the time of this writing are:
+
+- x86_64-mingw-gcc.cmake
+
+NOTE: Only x86_64 mingw cross compiling is available currently, in a future version we can consider adding more.  
+Please note that all dependent codec and external libraries should have been cross compiling already.
+
+The following example demonstrates use of the x86_64-mingw-gcc.cmake toolchain file on
+a x86\_64 Windows host:
+
+~~~
+    $ cmake ./ -DCMAKE_TOOLCHAIN_FILE=build/cmake/toolchains/x86_64-mingw-gcc.cmake
+    $ make
+~~~
+
+### Microsoft Visual Studio builds
+
+Building the IAMF library in Microsoft Visual Studio is supported. Visual
+Studio 2015 (14.0) solution has been provided.
+
+Open win32/VS2015/iamf.sln directly.
+
+## Testing the IAMF
+
+### 1. Build application
+
+iamfplayer is is a test application used to decode IAMF bitstream or mp4 file which is IAMF encapsulation.
```

Comment:
some typos here. How about this:

The iamfplayer is a test application to decode an IAMF bitstream or mp4 file with IAMF encapsulation.